pg/opm_core--2.5--2.6.sql

```diff
@@ -0,0 +1,60 @@
+-- This program is open source, licensed under the PostgreSQL License.
+-- For license terms, see the LICENSE file.
+--
+-- Copyright (C) 2012-2020: Open PostgreSQL Monitoring Development Group
+
+-- complain if script is sourced in psql, rather than via CREATE EXTENSION
+\echo Use "CREATE EXTENSION opm_core" to load this file. \quit
+
+ALTER TABLE public.api
+ALTER COLUMN proc TYPE text
+USING (pg_identify_object('pg_catalog.pg_proc'::regclass, proc, 0)).identity;
+
+CREATE OR REPLACE FUNCTION public.register_api(IN p_proc regprocedure,
+    OUT proc regprocedure, OUT registered boolean)
+LANGUAGE plpgsql STRICT VOLATILE LEAKPROOF
+SET search_path TO public
```

Comment:
Wouldn't it be better to set it to pg_catalog?

Reply:
done, thanks